task7/index.js

```diff
@@ -1,17 +1,22 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
 const div = document.getElementById("js-loading");
 const gif = document.createElement('img');
-gif.src = "loading-circle.gif";
 div.appendChild(gif);
+const loading = () => {
```

Comment:
(nits)You can omit the bracket, in this case.

Reply:
That's right.
This function has no sense.